tests/CatalogTests/Helpers/CatalogWriterHelperTests.cs

```diff
@@ -405,7 +405,8 @@ internal WritePackageDetailsToCatalogAsyncTest()
                     UtcNow.AddMinutes(-30),
                     UtcNow.AddMinutes(-15),
                     packageId: "a",
-                    packageVersion: "1.0.0");
+                    packageVersion: "1.0.0",
+                    packageFullVersion: "1.0.0");
```

Comment:
Perhaps make sure `packageVersion` and `packageFulVersion` differ?
e.g. by making `packageFullVersion` a SemVer2 version with metadata.

Reply:
Same for `Db2CatalogProjectionTests` where you added `packageFullVersion`.